setup.py

```diff
@@ -350,7 +356,9 @@ def find_version(filepath: str) -> str:
 def get_vllm_version() -> str:
     version = find_version(get_path("vllm", "version.py"))
 
-    if _is_cuda():
+    if _no_device():
+        version += "+empty"
```

Comment:
I was actually not sure if it is better to add "+empty" to the version or not.. WDYT?

Reply:
adding "+empty" looks good to me.